ci-operator/step-registry/openshift-qe/node-density/openshift-qe-node-density-commands.sh

```diff
@@ -15,14 +15,9 @@ ES_PASSWORD=$(cat "/secret/password")
 ES_USERNAME=$(cat "/secret/username")
 
 git clone https://github.com/cloud-bulldozer/e2e-benchmarking --depth=1
-pushd e2e-benchmarking/workloads/kube-burner
+pushd e2e-benchmarking/workloads/kube-burner-ocp-wrapper
+export EXTRA_FLAGS="--pods-per-node=$PODS_PER_NODE --pod-ready-threshold=$POD_READY_THRESHOLD"
 export WORKLOAD=node-density
-current_worker_count=$(oc get nodes --no-headers -l node-role.kubernetes.io/worker --output jsonpath="{.items[?(@.status.conditions[-1].type=='Ready')].status.conditions[-1].type}" | wc -w | xargs)
-export NODE_COUNT=$(($current_worker_count))
-export PODS_PER_NODE=230
-export POD_READY_THRESHOLD=180000ms
-export GEN_CSV=true
-export COMPARISON_CONFIG="clusterVersion.json podLatency.json containerMetrics.json kubelet.json etcd.json crio.json nodeMasters-max.json nodeWorkers.json"
 
 export CLEANUP_WHEN_FINISH=true
```

Comment:
sorry also this one is with the old kube-burner version I believe

Reply:
Done